root/user/UserProfile.js

```diff
@@ -0,0 +1,515 @@
+            <DescriptiveLink entity={user.area} />
+          </UserProfileProperty>
+        ) : null}
+
+        <UserProfileProperty name={l('Member since:')}>
+          {memberSince}
+        </UserProfileProperty>
+
+        {$c.user && (viewingOwnProfile || $c.user.is_account_admin) ? (
+          <UserProfileProperty name={l('Last login:')}>
+            {user.last_login_date
+              ? formatUserDate($c.user, user.last_login_date)
+              : l("Hasn't logged in yet")}
+          </UserProfileProperty>
+        ) : null}
+
+        {user.website ? (
+          <UserProfileProperty name={l('Homepage:')}>
+            {showBioAndURL ? (
+              <a href={user.website} rel="nofollow">
+                {user.website}
+              </a>
+            ) : (
+              <div className="deleted">
+                {exp.l(
+                  `This content is hidden to prevent spam.
+                  To view it, please {url|log in}.`,
+                  {url: '/account/login'},
+                )}
+              </div>
+            )}
+          </UserProfileProperty>
+        ) : null}
+
+        <UserProfileProperty name={l('Subscribers:')}>
+          {subscriberCount ? (
+            exp.l(
+              '{count} ({url|view list})',
+              {count: subscriberCount, url: `/user/${encodeURIComponent(user.name)}/subscribers`},
+            )
+          ) : (
+            l('0')
+          )}
+          {$c.user_exists && !viewingOwnProfile ? (
+            <>
+              {' '}
+              {bracketed(
+                subscribed ? (
+                  <a href={`/account/subscriptions/editor/remove?id=${encodeURIComponent(user.name)}`}>
+                    {l('unsubscribe')}
+                  </a>
+                ) : (
+                  <a href={`account/subscriptions/editor/add?id=${encodeURIComponent(user.name)}`}>
+                    {l('subscribe')}
+                  </a>
+                ),
+              )}
+            </>
+          ) : null}
+        </UserProfileProperty>
+
+        {user.biography ? (
+          <UserProfileProperty className="biography" name={l('Bio:')}>
+            {showBioAndURL ? (
+              <div dangerouslySetInnerHTML={{__html: user.biography}} />
+            ) : (
+              <div className="deleted">
+                {exp.l(
+                  `This content is hidden to prevent spam.
+                  To view it, please {url|log in}.`,
+                  {url: '/account/login'},
+                )}
+              </div>
+            )}
+          </UserProfileProperty>
+        ) : null}
+
+        {user.languages && user.languages.length > 0 ? (
+          <UserProfileProperty name={l('Languages:')}>
+            <ul className="inline">
+              {user.languages.map(language => (
+                <li key={language.language.id}>
+                  {`${l_languages(language.language.name)} (${l(language.fluency)})`}
```

Comment:
I don't think the `l()` call works there, because the fluency is lowercase while the translated strings in lib/MusicBrainz/Server/Form/User/EditProfile.pm have their first letter capitalized. That's probably why `l_fluency` in Perl uses `ucfirst`, first. You could import `lodash/capitalize` to do the same.

It'd also be nice if `bracketedText` was used for the parentheses. (If that's done, it'd not be necessary to embed any of it inside a template string.)